certified-operators/manifests/minio-operator.clusterserviceversion.yaml

```diff
@@ -503,7 +502,7 @@ spec:
             app.kubernetes.io/name: operator
           name: minio-operator
           spec:
-            replicas: 2
+            replicas: 1
```

Comment:
are we reducing the replicas to 1 now?

Reply:
Only in Openshift, we shipped with only one operator replica, AFAIK next deployments should have at least 2